pkg/dmapi/master_control.go

```diff
@@ -62,12 +62,12 @@ func (mc *defaultMasterControl) GetMasterClient(namespace string, dcName string,
 			return &masterClient{url: MasterClientURL(namespace, dcName, scheme), httpClient: &http.Client{Timeout: DefaultTimeout}}
 		}
 
-		return NewMasterClient(MasterClientURL(namespace, dcName, scheme), DefaultTimeout, tlsConfig)
+		return NewMasterClient(MasterClientURL(namespace, dcName, scheme), DefaultTimeout, tlsConfig, false)
 	}
 
 	key := masterClientKey(scheme, namespace, dcName)
 	if _, ok := mc.masterClients[key]; !ok {
-		mc.masterClients[key] = NewMasterClient(MasterClientURL(namespace, dcName, scheme), DefaultTimeout, nil)
+		mc.masterClients[key] = NewMasterClient(MasterClientURL(namespace, dcName, scheme), DefaultTimeout, nil, false)
```

Comment:
Why set to `false` here? The client will be used during each sync.

Reply:
Set to `false` so that don't disable `keep-alive`?